packages/client/katello-host-tools/katello-host-tools.spec

```
@@ -232,15 +247,15 @@ sed -i 's|bin/python$|bin/python3|' %{buildroot}%{plugins_dir}/tracer_upload.py
 
 #clean up tracer if its not being built
 %if %{build_tracer}
-rm %{buildroot}%{katello_libdir}/apt_tracer.py*
+# rm %{buildroot}%{katello_libdir}/apt_tracer.py*
```

Debugging purposes or this should actually be removed?

---

IF it's correct, it should be removed.  I'm not confident it's correct yet, so i left it in there as a note to confirm.

---

Based on the history of what I'm seeing in katello_host_tools, this theoretically should be renamed to deb_tracer.py instead of apt_tracer.py

---

Keeping it 1-1, it should be:
```suggestion
rm %{buildroot}%{katello_libdir}/deb_tracer.py*
```